symfony/plugins/orangehrmCorePlugin/Api/V2/Validator/Rules/Email.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Core\Api\V2\Validator\Rules;
+
+class Email extends AbstractRule
+{
+    private const EMAIL_REGEX = "/^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9]+)+$/";
```

Comment:
@super-chama please check this when you have time

Reply:
it's passed with **&a@gmail.com**
https://github.com/validatorjs/validator.js/blob/master/src/lib/isEmail.js

Reply:
this is the same regex that's used in the frontend in `rules.ts`. There is a different regex used for PIM CSV data import `/^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,3})$/i`

Reply:
![image](https://user-images.githubusercontent.com/12880059/162364632-6504d15f-9ab1-4890-a807-71af886940e2.png)

Looks like having & infront of mail address doesn't make it syntactically invalid. since we are only checking the syntax here i'm good with current regex.